fire/SFMainMod.F90

```diff
@@ -94,13 +103,20 @@ subroutine fire_model( currentSite, bc_in)
     if( hlm_use_spitfire == itrue )then
        call fire_danger_index(currentSite, bc_in)
        call wind_effect(currentSite, bc_in) 
-       call charecteristics_of_fuel(currentSite)
-       call rate_of_spread(currentSite)
+       call characteristics_of_fuel(currentSite, MEF, fuel_moisture)
+       call rate_of_spread(currentSite, MEF, fuel_moisture)
        call ground_fuel_consumption(currentSite)
        call fire_intensity(currentSite)
        call area_burnt(currentSite)
        call crown_scorching(currentSite)
        call crown_damage(currentSite)
+       ! Begin: Repeat calls to calculate effects of active crown fire
```

Comment:
Currently this is determined by `currentCohort%active_crown_fire_flg = 1` which cannot be checked here because this subroutine runs at the Site level; however, we could sum all the `active_crown_fire_flg = 1` and if that sum excedes 0, then we can execute these repeat calls.